lib/factory_base.js

```diff
@@ -44,14 +44,13 @@ class FactoryBase {
     } else {
       if (!this._jsonld_cache) this._jsonld_cache = {}
 
-      let content = null
-      try {
-        content = request('GET', `https://raw.githubusercontent.com/NYPL/nypl-core/${coreVersion}/vocabularies/json-ld/${filename}.json`).getBody()
-      } catch (e) {
-        throw new FactoryBase.MappingNameError(`Unrecognized mapping file: ${filename}`)
-      }
+      const url = `https://raw.githubusercontent.com/NYPL/nypl-core/${coreVersion}/vocabularies/json-ld/${filename}.json`
+      this._jsonld_cache[filename] = axios.get(url)
```

Comment:
can this be in try/catch blocks instead of then().catch()? if we're going with async/await i think its more consistent that way.

Reply:
Restructured to use async/await.